src/main/main.ts

```diff
@@ -90,10 +94,30 @@ const createWindow = async () => {
     } else {
       mainWindow.show();
     }
+
+    pyServer = startServer();
+
+    if (pyServer !== null && pyServer.stderr !== null) {
+      pyServer.stderr.once('data', (data) => {
+        console.log(data.toString()); // prints address that the server is running on. TODO: remove before it is electron app is built
```

Comment:
I guess we can leave this in for now, but might want to check NODE_ENV before logging so we don't log in prod

Reply:
good idea